www/data_module/gulpfile.coffee

```diff
@@ -51,9 +51,9 @@ gulp.task 'build', ['clean'], ->
     gulp.src files.coffee
```

Comment:
Please use guanlecoja for building.

guanlecoja-ui has exactly the same build needs, so please use the same methodology.